cli/utils.go

```diff
@@ -16,27 +17,89 @@ func getValidArgsString(md *plugin.GetMetadataResponse) string {
 
 // getValidArgs returns list of valid arguments depending upon the plugin capabilities
 func getValidArgs(md *plugin.GetMetadataResponse) []string {
-	opts := []string{
-		"get-plugin-metadata", "version",
+	opts := []plugin.Command{
+		plugin.CommandGetMetadata, plugin.Version,
 	}
 
 	if slices.Contains(md.Capabilities, plugin.CapabilitySignatureGenerator) {
-		opts = append(opts, "generate-signature", "describe-key")
+		opts = append(opts, plugin.CommandGenerateSignature, plugin.CommandDescribeKey)
 	}
 
 	if slices.Contains(md.Capabilities, plugin.CapabilityEnvelopeGenerator) {
-		opts = append(opts, "generate-envelope")
+		opts = append(opts, plugin.CommandGenerateEnvelope)
 	}
 
 	if slices.Contains(md.Capabilities, plugin.CapabilityTrustedIdentityVerifier) || slices.Contains(md.Capabilities, plugin.CapabilityRevocationCheckVerifier) {
-		opts = append(opts, "verify-signature")
+		opts = append(opts, plugin.CommandVerifySignature)
 	}
-	sort.Strings(opts)
-	return opts
+	// convert &opts (type *[]Command) to *[]string via unsafe.Pointer, then deref
+	stringOpts := *(*[]string)(unsafe.Pointer(&opts))
```

Comment:
Can we avoid using unsafe.Pointer here?
For example:
```go
opts := []string{
		string(plugin.CommandGetMetadata), string(plugin.Version),
	}
```

Reply:
Yep that's possible but reduces readability and performance. Is ther4e any specific concern with  unsafe.Pointer as command is basically an alias for string?